### ISSUE TITLE
Undefined array key "hasSubPages"

### ISSUE THREAD

**Issue by grischpel** [FluidTYPO3/vhs#1822]
(1/1) #1476107295 TYPO3\CMS\Core\Error\Exception
PHP Warning: Undefined array key "hasSubPages" in /public/typo3conf/ext/vhs/Classes/ViewHelpers/Menu/AbstractMenuViewHelper.php line 314

**Comment by DanYouCan** [comment]
With TYPO3 V11.5.21 PHP8.1.13 I got same php error:
Core: Error handler (FE): PHP Warning: Undefined array key "hasSubPages" in /public/typo3conf/ext/vhs/Classes/ViewHelpers/Menu/AbstractMenuViewHelper.php line 314

**Comment by ste101** [comment]
I change line 314 in AbstractMenuViewHelper.php to this
```
if (($page['active'] || $expandAll) && ($page['hasSubPages'] ?? '') && $level < $levels) {
```

**Comment by fixpunkt-systeme** [comment]
Even when the Configuration Presets "Live" is selected, the Log-file contains this Warning.
Therefore this bug should be fixed.

**Comment by NamelessCoder** [comment]
Has been solved in `dev-development` since c78039646dc1dd43a4c070bb060a17c2102be777
